wasm.md

```diff
@@ -1012,12 +1004,12 @@ Sort `Signedness` is defined in module `BYTES`.
 
 ```k
     syntax Instr ::= "load" "{" IValType Int Int Signedness"}"
- // ----------------------------------------------------------
+                   | IValType "." LoadOp Int
```

Comment:
Why is this turned into an `Instr` instead of a `PlainInstr`?

Reply:
Because `PlainInstr` is only for instructions that may appear in source text. This is an administrative instruction. It may be better for format it as such, or maybe use a function instead.